crates/graphql_artifact_generation/src/iso_overload_file.rs

```diff
@@ -159,20 +177,13 @@ export function iso(_isographLiteralText: string):
 {\n",
     );
 
-    content.push_str(match on_missing_babel_transform {
-        OptionalValidationLevel::Error => {
+    content.push_str(match no_babel_transform {
+        false => {
 "  throw new Error('iso: Unexpected invocation at runtime. Either the Babel transform ' +
       'was not set up, or it failed to identify this call site. Make sure it ' +
       'is being used verbatim as `iso`.');"
         }
-        OptionalValidationLevel::Warn => {
-
-            "  console.warn('iso: Unexpected invocation at runtime. Either the Babel transform ' +
-      'was not set up, or it failed to identify this call site. Make sure it ' +
-      'is being used verbatim as `iso`.');
-  return (clientFieldResolver: any) => clientFieldResolver;"
```

Comment:
can we modify this error message too? E.g. add "If you cannot use the babel transform, set `options.no_babel_transform` to `true` in your Isograph config."